tests/integration_tests/Gopkg.toml

```diff
@@ -0,0 +1,38 @@
+# Gopkg.toml example
```

Comment:
I'm curious why you chose Gopkg as opposed to Gomod here.

Reply:
I need to improve my understanding more in other to respond to this question. I mainly followed another lambda example. Will circle back to this after my docker changes.

Reply:
OK - I went with go.mod instead and used `replace` to import the current local code. I'm super new with Go so any feedback is appreciated